cli/tsc/97_ts_host.js

```diff
@@ -714,7 +728,33 @@ export const host = {
     }
     return scriptSnapshot;
   },
+  // getModuleResolutionCache() {
+  //   return resolutionCache;
+  // }
 };
+// resolutionCache = ts.createResolutionCache(host, host.getCurrentDirectory(), true);
+const excluded = new Set([
+  "getScriptVersion",
+  "fileExists",
+  "getScriptSnapshot",
+  "getCompilationSettings",
+  "getCurrentDirectory",
+  "useCaseSensitiveFileNames",
+  "getModuleSpecifierCache",
+  "getGlobalTypingsCacheLocation",
+  "getSourceFile",
```

Comment:
What's the reason for excluding these? Too noisy?

Reply:
yeah way too noisy, like thousands of calls per tsc request